sdk/tests/iot/adu/test_az_iot_adu.c

```diff
@@ -925,6 +998,35 @@ static void test_az_iot_adu_client_parse_service_properties_multiple_file_url_va
   assert_int_equal(request.file_urls_count, 2);
 }
 
+static void test_az_iot_adu_client_parse_service_properties_too_many_file_url_values_succeed(
```

Comment:
Should this be a fail test since it's checking for an error at the end?
```suggestion
static void test_az_iot_adu_client_parse_service_properties_too_many_file_url_values_fail(
```

Reply:
Crap yup knew I forgot something